pyproject.toml

```diff
@@ -43,6 +43,7 @@ dependencies = [
     "scikit-learn-extra>=0.3.0",
     "scipy>=1.10.1",
     "setuptools-scm>=7.1.0",
+    "tabulate>=0.9.0",
```

Comment:
While not terrible: have you checked if we could use some built-in alternatives from packages that we already use (e.g. pandas itself)? Always a bit hesitant to add more dependencies unless there are really needed.

Reply:
Okay I will check if I can find a better solution without using tabulate.

Reply:
i would caution against adding additional dependencies and make exceptions only if something is strictly necessary, having an additional dependency for pretty printing does not seem justified imo